vizro-ai/docs/pages/tutorials/quickstart.md

```diff
@@ -65,7 +65,12 @@ To learn how to customize the `VizroAI` class, check out the guide on [how to cu
 Finally, we call the `plot()` method with our English language instruction, to generate the visualization:
 
 ```python
-vizro_ai.plot(df, "create a line graph for GDP per capita since 1950 for each continent. Mark the x axis as Year, y axis as GDP Per Cap and don't include a title")
+vizro_ai.plot(
+    df,
+    """create a line graph for GDP per capita since 1950 for
+    each continent. Mark the x axis as Year, y axis as GDP Per Cap
+    and don't include a title. Make sure to take average over continent.""",
+)
```

Comment:
I can replicate the difference you mentioned in description. But to me it's not a performance downgrade. The chart now created is not wrong, it's just colored by continent. 

Plus, if just adding "average" to the sentence, it knows to aggregate.